src/renderer/components/kube-object-menu/dependencies/static-kube-object-menu-items.ts

```diff
@@ -37,11 +36,4 @@ export const staticKubeObjectMenuItems = [
       MenuItem: ReplicaSetMenu,
     },
   },
-  {
```

Comment:
I changed this single `KubeObjectMenu` registration into a `KubeObjectHandler` registration to show in this PR how it can be done.